src/interactions/interaction.ts

```diff
@@ -11,8 +11,53 @@ module Plottable {
      */
     protected _componentToListenTo: Component;
 
-    public _anchor(component: Component) {
+    private _anchorCallback = (component: Component) => this._anchor(component);
+
+    private _isAnchored: Boolean;
+
+    /* tslint:disable: */
+    // tslint disable does not really work
+    protected _anchor(component: Component) {
+      this._isAnchored = true;
+    }
+
+    protected _unanchor() {
+      this._isAnchored = false;
+    }
+    /* tslint:enable */
```

Comment:
Looks like a previous commit had the blocks empty...? I tried pulling the branch in its current state and it worked fine without the tslint stuff.